mail_debrand/models/ir_qweb.py

```diff
@@ -0,0 +1,21 @@
+from odoo.addons.base.models.qweb import QWeb
+from odoo import models
+
+
+class IrQWeb(models.AbstractModel, QWeb):
+    _inherit = 'ir.qweb'
+
+    def render(self, template, values=None, **options):
+        result = super(IrQWeb, self).render(template, values=values, **options)
+        lang = values.get("lang", False)
+        is_mail = False
+        # search for mail.message object in values
+        for value in values.values():
+            if isinstance(value, models.Model) and value._name == "mail.message":
+                is_mail = True
+                break
+        if lang and is_mail:
```

Comment:
Checking for `lang` and any `mail.message` record in values is a very weak condition to be sure that an email is being sent for basically the same reasons I wrote in https://github.com/OCA/social/pull/736#discussion_r694986881